libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthPreferences.h

```diff
@@ -106,5 +106,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic,copy) NSString *appDisplayName;
 
+/**
+ Whether the app is configured to require certificate-based authentication. (RequireCertAuth)
+ */
+@property (nonatomic, assign) BOOL requireCertificateAuthentication;
```

Comment:
Should we call this the same name as above, i.e. <i>requireBrowserAuth</i>, or something along those lines?